### PR TITLE
Update the ctest profile usage

### PR DESCRIPTION
**Pull request by draftcode**
Wait until https://github.com/launchableinc/cli/pull/303 is released.

### CODE REVIEW

**Comment by draftcode** [comment]
I want to check with you on the flag names. Do they look OK to you?

File: docs/resources/integrations/ctest.md [hunk 1]
```diff
@@ -24,9 +24,54 @@ You might need to take extra steps to make sure that `launchable record tests` a
...
+and pass that to `launchable subset`:
+
+```bash
+# --show-only=json-v1 option outputs test list as JSON
+ctest --show-only=json-v1 > test_list.json
+launchable subset \
+    --build <BUILD NAME> \
+    --confidence <TARGET> \
+    --output-regex-files \
+    --output-regex-file-dir=subsets \
```
Comment (awilkes) [inline docs/resources/integrations/ctest.md:43]: How about `--output-regex-files-dir`? (+`s`)

**Review by awilkes — APPROVED** [review]

File: docs/resources/integrations/ctest.md (hunk 1 above)
Comment (draftcode) [reply docs/resources/integrations/ctest.md]: Done.

**Comment by awilkes** [comment]
Thanks @draftcode!

**Pull request merged**